src/DssSpell.sol

```diff
@@ -29,6 +33,10 @@ contract DssSpellAction is DssAction {
         return false;
     }
 
+    // Approve DAO resolution hash QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w
+    // Note: by the previous convention it should be a comma-separated list of DAO resolutions IPFS hashes
+    string public constant dao_resolutions = "QmVtqkYtx61wEeM5Hb92dGA3TMZ9F1Z5WDSNwcszqxiF1w"; // TODO: update  
+
```

Comment:
I couldn't find this on the sheet. Could you please double-check?

Reply:
It's indeed not present, removed it.